python/chronos/test/bigdl/chronos/data/test_tsdataset.py

```diff
@@ -1092,3 +1090,101 @@ def test_cycle_length_est(self):
                                        dt_col='datetime')
         with pytest.raises(RuntimeError):
             tsdata.get_cycle_length(aggregate='min', top_k=3)
+
+    def test_is_predict_for_roll_and_numpy(self):
+        df = get_ts_df()
+        horizon = random.randint(1, 10)
+        lookback = random.randint(1, 20)
+
+        tsdata = TSDataset.from_pandas(df, dt_col="datetime", target_col="value",
+                                       extra_feature_col=["extra feature"], id_col="id")
+
+        with pytest.raises(RuntimeError):
+            tsdata.to_torch_data_loader(roll=False)
+
+        # for BaseForecaster
+        data = tsdata.roll(lookback=lookback, horizon=horizon, is_predict=True).to_numpy()
+        assert not isinstance(data, (list, tuple))
+
+        tsdata = TSDataset.from_pandas(df, dt_col="datetime", target_col="value",
+                                       extra_feature_col=["extra feature"], id_col="id")
+
+        with pytest.raises(RuntimeError):
+            tsdata.to_torch_data_loader(roll=False)
```

Comment:
same callings in 1099-1103 and 1109-1113 for 2 times

Reply:
maybe no need to test `to_torch_data_loader` in this UT?

Reply:
will remove these tests.